client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1918,6 +1919,21 @@ Undocumented.prototype.updateImporter = function( siteId, importerStatus ) {
 	} );
 };
 
+Undocumented.prototype.importWithSiteImporter = function(
+	siteId,
+	importerStatus,
+	params,
+	targetUrl
+) {
+	debug( `/sites/${ siteId }/site-importer/import-site?${ stringify( params ) }` );
+
+	return this.wpcom.req.post( {
+		path: `/sites/${ siteId }/site-importer/import-site?${ stringify( params ) }`,
+		apiNamespace: 'wpcom/v2',
+		formData: [ [ 'import_status', JSON.stringify( importerStatus ) ], [ 'site_url', targetUrl ] ],
```

Comment:
Personally I find this bit easier to read with the `formData` elements on separate lines, but that's subjective I guess.

Reply:
I did update that code, but the auto code formatter in the git commit hooks reformats it to what we see above :(